src/main/java/com/conveyal/taui/persistence/MongoMap.java

```diff
@@ -73,18 +75,21 @@ public V get(Object key) {
     }
 
     public Collection<V> findAllForRequest(Request req) {
-        return find(QueryBuilder.start("accessGroup").is(req.attribute("accessGroup")).get());
+        return find(QueryBuilder.start("accessGroup").is(req.attribute("accessGroup")).get()).toArray();
     }
 
     public Collection<V> findPermitted(DBObject query, String accessGroup) {
         return find(QueryBuilder.start().and(
                 query,
                 QueryBuilder.start("accessGroup").is(accessGroup).get()
-        ).get());
+        ).get()).toArray();
     }
 
-    public Collection<V> find(DBObject query) {
-        return wrappedCollection.find(query).toArray();
+    /**
+     * All Models have a createdAt field. By default, sort by that field.
+     */
+    public DBCursor<V> find(DBObject query) {
+        return wrappedCollection.find(query).sort(DBSort.desc("createdAt"));
```

Comment:
Why did you factor the `toArray()` call out to the callers?

Reply:
So that we can use the Cursor object (for more Mongo queries like `limit`) if we want to.